pkgs/applications/office/gnucash/default.nix

```
@@ -97,12 +98,29 @@ stdenv.mkDerivation rec {
   enableParallelChecking = true;
   checkTarget = "check";
 
+  passthru.gnucash-docs = stdenv.mkDerivation {
```

```suggestion
  passthru.docs = stdenv.mkDerivation {
```

After all, calling `gnucash.gnucash-docs` is too much redundancy.